spec/support/integration_spec_helper.rb

```diff
@@ -2,11 +2,11 @@
 
 module IntegrationSpecHelper
   SAMPLE_DOCUMENT_ATTRIBUTES = {
-    "title" => "TITLE1",
-    "description" => "DESCRIPTION",
-    "format" => "local_transaction",
-    "link" => "/URL",
-    "_type" => "edition",
+      "title" => "TITLE1",
```

Comment:
Same comment about ditching the whitespace as before - if nothing's changed we don't want to obscure the git history.